src/resmom_win/mom_interactive_shell.c

```diff
@@ -100,9 +102,12 @@ main(int argc, char *argv[])
 	 * Create std pipes and wait for qsub to connect to these pipes
 	 */
 	strncpy(pipename_append, momjobid, PIPENAME_MAX_LENGTH - 1);
-	if (create_std_pipes(&si, pipename_append, 1) != 0)
+	if ((rc = create_std_pipes(&si, pipename_append, 1)) != 0) {
+		fprintf(stderr, "mom_shell: Failed to create pipe with error=%lu\n", rc);
```

Comment:
Don't we need to use log_err in this file like others?

Reply:
For interactive jobs, the rest of the error messages in this file are logged using fprintf to stderr. So I logged this one the same way.